### PR TITLE
Follow up for initial SATB barrier changes

### PR DESCRIPTION
**Pull request by ebadyano**
Follow up for 2717. Removing redundant parameters and function calls since openj9 changes are merged.

Signed-off-by: Evgenia Badyanova <evgeniab@ca.ibm.com>

### CODE REVIEW

**Comment by ebadyano** [comment]
@amicic @dmitripivkine Could you please take a look?

**Comment by dmitripivkine** [comment]
Should you do something with "include "GCExtensions.hpp"" in WorkPacketsSATB.cpp ? I guess use GCExtensionsBase.hpp instead...

**Comment by amicic** [comment]
looks good

File: gc/base/standard/WorkPacketsSATB.cpp [hunk 1]
```diff
@@ -64,17 +63,10 @@ MM_WorkPacketsSATB::initialize(MM_EnvironmentBase *env)
 		return false;
 	}
 
-	if (0 == MM_GCExtensions::getExtensions(_extensions)->overflowCacheCount) {
-		/* If the user has not specified a value for overflowCacheCount
```
Comment (charliegracie) [inline gc/base/standard/WorkPacketsSATB.cpp]: where did this code move?

**Comment by charliegracie** [comment]
@genie-omr build all

File: gc/base/standard/WorkPacketsSATB.cpp (hunk 1 above)
Comment (ebadyano) [reply gc/base/standard/WorkPacketsSATB.cpp]: For WorkPacketsSATB we decided to use OverflowStandard which doesn't use overflowCacheCount, and WorkPacketsRealtime (subclass of WorkPacketsSATB and used in metronome)  still uses IncrementalOverflow and sets overflowCacheCount (in openj9).

**Comment by charliegracie** [comment]
@dmitripivkine are you ok with the changes now?

**Comment by dmitripivkine** [comment]
looks good for me

**Comment by charliegracie** [comment]
@genie-omr build all

**Pull request merged**